src/backend/src/services/projects.services.ts

```diff
@@ -949,17 +949,8 @@ export default class ProjectsService {
     });
 
     if (!materialType) throw new NotFoundException('Material Type', materialTypeId);
-    if (materialType.dateDeleted) throw new DeletedException('Material Type', materialTypeId);
-
-    const deletedMaterialType = await prisma.material_Type.update({
-      where: {
-        name: materialTypeId
-      },
-      data: {
-        dateDeleted: new Date()
-      }
-    });
 
+    const deletedMaterialType = await prisma.material_Type.delete({ where: { name: materialTypeId } });
```

Comment:
oops, should've mentioned this on the ticket, but we also need to make sure that the material type is not currently associated with any materials and throw an error if it is, because otherwise prisma will get mad at us.

material types should only really be deleted if they have a typo (in which case they shouldn't be associated with any materials)